src/Components/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { WishlistCardItem } from "./WishlistCardItem";
+import { useStateProvider } from "../../Context/StateContext/state-provider";
+
+export const Wishlist = () => {
+  const { userState } = useStateProvider();
+  return (
+    <div>
+      <h3 className="text-center">My Wishlist </h3>
+
+      <div className="spacer-3rem"></div>
+
+      <div className="grid-4-column-layout">
+        {userState.wishlistItems?.map((prod) => {
+          return <WishlistCardItem prod={prod} key={prod.id} />;
+        })}
```

Comment:
No need to explicitly mention return, you can remove the curly brackets and directly pass the `WishlistCardItem` component inside your map function